tests/metrics/lib/common.bash

```diff
@@ -201,9 +197,12 @@ function kill_processes_before_start()
 	CTR_PROCS=$(sudo "${CTR_EXE}" t list -q)
 	[[ -n "${CTR_PROCS}" ]] && clean_env_ctr
 
-	kill_kata_components
+	restart_containerd_service
```

Comment:
Do need containerd restart twice? I see you are restarting on line 200 and later on line 205 as well?

Reply:
@amshinde containerd just need to be restarted once, thanks.